string time_str

float32 time_int

float32 SystemCal
float32 GyroCal
float32 AccelCal
float32 MagCal

float32 PosX
float32 PosY
float32 PosZ

float32 MagX
float32 MagY
float32 MagZ

float32 GyroX
float32 GyroY
float32 GyroZ

float32 QuatX
float32 QuatY
float32 QuatZ
float32 QuatW

float32 LinAccX
float32 LinAccY
float32 LinAccZ

float32 EulerX
float32 EulerY
float32 EulerZ

float32 AbsAccX
float32 AbsAccY
float32 AbsAccZ